CHANGELOG.md

```diff
@@ -1,6 +1,35 @@
 Changelog
 =========
 
+v6.0.0-alpha.17 - 2017-03-08
+----------------------------
+- Update PSRP client libraries for Linux and Mac.
+  - We now support customer configurations for Office 365 interaction, as well as NTLM authentication for WSMan-based remoting from Linux (more information [here](https://github.com/PowerShell/psl-omi-provider/releases/tag/v1.0.0.18)). (#3271)
+- We now support remote step-in debugging for `Invoke-Command -ComputerName`. (#3015)
+- Use prettier formatter with `ConvertTo-Json` output. (#2787) (Thanks to @kittholland!)
```

Comment:
There should be a space between the mention and `!` or you should make it a link manually.